config-5.1.lua

```diff
@@ -1,3 +1,12 @@
 rocks_trees = {
-   { name = [[openresty]], root = [[/usr/local/openresty/luajit]] }
+   {
+      name = [[openresty]],
+      root = [[/usr/local/openresty/luajit]],
+      bin_dir = [[/opt/app-root/bin]]
```

Comment:
Probably would be better to install it somewhere in `/usr/local/openresty` and give that folder a write permission like for the lib and external deps.